src/normalizer.ts

```diff
@@ -222,6 +223,29 @@ rules.set('Transform const to singleton enum', schema => {
   }
 })
 
+rules.set('Transform nullable to null type', schema => {
+  if (schema.nullable !== true) {
+    return
+  }
+
+  delete schema.nullable
+
+  // schema.const has already been converted to a single-value enum by rule "Transform const to singleton enum", so it
+  // does not need to be handled here.
+  if (schema.enum) {
+    if (!schema.enum.includes(null)) {
+      schema.enum.push(null)
+      log('yellow', 'normalizer', 'enum should include "null" when schema is nullable', schema)
```

Comment:
1. Need to remove quotes around `null` in the message since the value in the enum should not have quotes
2. Unsure whether this is the level of log desired, or if it would be better to use `error()` from utils.ts